operator/main.go

```diff
@@ -275,6 +278,8 @@ func defineFlagVar() *FlagVar {
 		"The resources.limits.memory for skr webhook.")
 	flag.StringVar(&flagVar.skrWebhookCPULimits, "skr-webhook-cpu-limits", "0.1",
 		"The resources.limits.cpu for skr webhook.")
+	flag.BoolVar(&flagVar.enableWebhookPreInstallCheck, "enable-webhook-pre-install-check", false,
```

Comment:
is this webhook for the skr? if so i think we should rename it to contain skr. Also looking at flag wars, we can maybe simplify to "enable-watcher" since it completely disables all watcher logics.

Reply:
This flag was used to toggle a check done before installing the chart to prevent overriding it if the same chart resources are installed already when the `enable-watcher` flag is true but I think we do not need it anymore. I am going to remove it 👍